payments-core/src/test/java/com/stripe/android/payments/PaymentIntentFlowResultProcessorTest.kt

```diff
@@ -100,46 +100,23 @@ internal class PaymentIntentFlowResultProcessorTest {
         }
 
     @Test
-    fun `no refresh when user cancels the payment`() =
-        runTest {
-            whenever(mockStripeRepository.retrievePaymentIntent(any(), any(), any())).thenReturn(
-                Result.success(PaymentIntentFixtures.PI_REQUIRES_WECHAT_PAY_AUTHORIZE)
-            )
-            whenever(mockStripeRepository.refreshPaymentIntent(any(), any())).thenReturn(
-                Result.success(PaymentIntentFixtures.PI_REFRESH_RESPONSE_WECHAT_PAY_SUCCESS)
-            )
-
-            val clientSecret = "pi_3JkCxKBNJ02ErVOj0kNqBMAZ_secret_bC6oXqo976LFM06Z9rlhmzUQq"
-            val requestOptions = ApiRequest.Options(apiKey = ApiKeyFixtures.FAKE_PUBLISHABLE_KEY)
-
-            val paymentIntentResult = processor.processResult(
-                PaymentFlowResult.Unvalidated(
-                    clientSecret = clientSecret,
-                    flowOutcome = StripeIntentResult.Outcome.CANCELED
-                )
-            ).getOrThrow()
+    fun `no refresh when user cancels the payment`() = runTest {
```

Comment:
Reducing what this test actually checks, so that it aligns more with with test name. For instance, we no longer test the result returned from the processor.